src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -181,6 +182,56 @@ public static void CopyToShorterArray()
             Assert.Equal<int>(expected, dst);  // CopyTo() checks for sufficient space before doing any copying.
         }
 
+        private static void CopyTo_LargeSize_TestHelper(long BufferSize)
+        {
+            int GuidCount = (int)(BufferSize / Unsafe.SizeOf<Guid>());
+
+            unsafe
+            {
+                bool allocatedFirst = AllocationHelper.TryAllocNative((IntPtr)BufferSize, out IntPtr memBlockFirst);
+                bool allocatedSecond = AllocationHelper.TryAllocNative((IntPtr)BufferSize, out IntPtr memBlockSecond);
+
+                try
+                {
+                    ref Guid memoryFirst = ref Unsafe.AsRef<Guid>(memBlockFirst.ToPointer());
+                    var spanFirst = new Span<Guid>(memBlockFirst.ToPointer(), GuidCount);
+
+                    ref Guid memorySecond = ref Unsafe.AsRef<Guid>(memBlockSecond.ToPointer());
+                    var spanSecond = new Span<Guid>(memBlockSecond.ToPointer(), GuidCount);
+
+                    Guid theGuid = Guid.Parse("900DBAD9-00DB-AD90-00DB-AD900DBADBAD");
```

Comment:
I just copied the kernel of Overflow.cs test. I thought that Guid was used there because I felt that Guid 128 bits will be a faster copy than int 32 bits...